managed-cluster-config-services/managed-cluser-config.yaml

```diff
@@ -0,0 +1,6 @@
+services:
+- hash: none
+  name: managed-cluster-config
+  path: /00-osd-manged-cluster-config.selectorsyncset.yaml 
```

Comment:
`manged` typo?

and also I don't seem to see that file in the repo?

Reply:
It is a typo. The file is generated by the make command

Reply:
unfortunately that doesn't work, it has to be precommited

cc @jharrington22

Reply:
cc @jewzaam

Reply:
I've updated the file path and committed it to the other repo